File: Front/ProEventos-App/src/app/models/palestrante.ts
import { RedeSocial } from "./rede-social";

export interface Palestrante {
    Id: number;
    Nome: string;
    MiniCurriculo: string;
    ImagemURL: string; 
    Telefone: string; 
    Email: string; 
    RedeSocial: RedeSocial[]; 
    PalestranteEvento: Palestrante[];  
}

File: Back/ProEvento.API/ProEvento.Domain/Entities/Palestrante.cs
﻿namespace ProEvento.Domain.Models;

public class Palestrante
{
    public int Id { get; set; }
    public string Nome { get; set; }
    public string MiniCurriculo { get; set; }
    public string ImagemURL { get; set; }
    public string Telefone { get; set; }
    public string Email { get; set; }
    public IEnumerable<RedeSocial> RedeSociais { get; set; }
    public IEnumerable<PalestranteEvento> PalestrantesEventos { get; set; }
}
